.env.example

```diff
@@ -21,6 +21,9 @@ INFLUXDB_ADMIN_PASSWORD=admin
 INFLUXDB_USER=api
 INFLUXDB_USER_PASSWORD=secretpassword
 
+# Country of platform deployment and usage. Should be specified following ISO Alpha-2 format
+PLATFORM_COUNTRY_CODE=TH
```

Comment:
Don't we have this already defined in /api/configuration?

Reply:
Good point. Yes, we have it.
I will have to force push the branch in order to remove the breaking_change commit